client/src/components/Atoms/CSVForm.tsx

```diff
@@ -95,7 +95,8 @@ const CSVFile: React.FC<IProps> = ({
                 {description}
                 {values.cvrFileType === CvrFileType.HART &&
                   ' ' +
-                    'For Hart, you can also provide an optional scanned ballot information CSV. ' +
+                    'For Hart, you can provide one ZIP file or multiple, one for each tabulator. ' +
+                    'You can also provide an optional scanned ballot information CSV. ' +
```

Comment:
I know this copy is getting a bit long:

![copy](https://user-images.githubusercontent.com/12616928/200208016-e4b1700e-d567-4253-bf46-d815d25e4782.png)

Happy to add a line break between the generic copy and the Hart-specific copy!